website/source/docs/commands/alloc/exec.html.md.erb

```diff
@@ -0,0 +1,83 @@
+---
+layout: "docs"
+page_title: "Commands: alloc exec"
+sidebar_current: "docs-commands-alloc-exec"
+description: >
+Stream the exec of a task.
```

Comment:
Can do better than that haha

Reply:
Oops - I used logs as a template with `%s/logs/exec/g` and missed this one.